site/_includes/layouts/namespace-reference.njk

```diff
@@ -51,7 +51,8 @@
 
         {% if f.platformAppsOnly %}
           <div class="aside aside--warning">
-            This API is part of the deprecated Chrome Apps platform.
+            This API is part of the Chrome Apps platform, which was deprecated in 2020.
+            It is supported only for ChromeOS until Jan 2025.
```

Comment:
```suggestion
            It is only supported only for ChromeOS enterprise and education customers until Jan 2025.
```

Should we clarify this? Took the info from: https://blog.chromium.org/2021/10/extending-chrome-app-support-on-chrome.html

Reply:
Thanks for the callout Sebastian. @jpmedley, not sure if you were already planning this but we might want to tweak the wording? I don't think it reads great now.

Reply:
What do you want it to say?

Reply:
@oliverdunk have you got any suggestions? 

Update lgtm from my side with the explicit mention of education and enterprise.

Reply:
How about:

>  This API is part of the Chrome Apps platform, which was deprecated in 2020. It remains supported for Enterprise and Education customers on ChromeOS until at least Jan 2025.

The "at least" is based on https://blog.chromium.org/2021/10/extending-chrome-app-support-on-chrome.html unless there was a more recent update I've missed.

Reply:
Done.